src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,84 @@
-export const Pagination = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const paginationItemsCount = Math.ceil(total / perPage);
+  const next = currentPage >= paginationItemsCount;
```

Comment:
The variable 'next' is misleading because it suggests a boolean value indicating the availability of a next page, but its value is the opposite of what its name suggests. Consider renaming it to 'isLastPage' or 'noNextPage' to improve readability.